portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -44,10 +45,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     int numComments = Integer.parseInt(request.getParameter("numComments"));    
-    int count = 0;
     List<Comment> entries = new ArrayList<>();
-
-    UserService userService = UserServiceFactory.getUserService();  
+    UserService userService = UserServiceFactory.getUserService();
+    int count = 0;
```

Comment:
Is count ever used?

Reply:
yes, it's used in the if statement below and incremented before the end of the for loop

Reply:
Oops, my mistake. Resolving.